share_app/src/main/java/org/odk/share/activities/MainActivity.java

```diff
@@ -138,9 +138,11 @@ protected String getSortingOrderKey() {
     @Override
     public void onLoadFinished(@NonNull Loader<Cursor> loader, Cursor cursor) {
         formAdapter.changeCursor(cursor);
-        if (cursor != null && !cursor.isClosed()) {
-            setEmptyViewVisibility(cursor.getCount());
-        }
+        if (cursor != null) {
```

Comment:
It could be like
```
if (cursor != null && !cursor.isClosed()) {
    setEmptyViewVisibility(cursor.getCount());
    return;
}
setEmptyViewVisibility(0);
```